R/MappedDataBoxplot.R

```diff
@@ -0,0 +1,133 @@
+# Boxplots ----------------
+#' @title object to map data for boxplots
+#' @description  R6 class for mapping variable to `data`
+#' @export
+#' @family MappedData classes
+MappedDataBoxplot <- R6::R6Class( # nolint
+  "MappedDataBoxplot",
+  inherit = MappedData,
+  public = list(
+    #' @field xscale scale of x axis
+    xscale = NULL,
+    #' @field xscale.args arguments for scale of x axis
+    xscale.args = NULL,
+    #' @field hasXmapping Flag, if TRUE x is mapped
+    hasXmapping = NULL,
+    #'
+    #' @param data data.frame used for mapping
+    #' @param mapping list of aesthetic mappings
+    #' @param groupAesthetics vector of aesthetics, which are used for columns mapped with aesthetic `groupby`
+    #' @param direction direction of plot either "x" or "y"
+    #' @param isObserved Flag if TRUE mappings mdv, lloq, error and error_relative are evaluated
```

Comment:
I never saw `Flag` used  in R packages or OSPS R packages to describe a `boolean`. I can understand It but I'm not sure the users are really familiar with this.